gcloud-java-contrib/README.md

```diff
@@ -7,6 +7,12 @@ Contents
 --------
 
  * [gcloud-java-nio](./gcloud-java-nio/): NIO Filesystem Provider for Google Cloud Storage.
+ * [gcloud-java-nio-example](./gcloud-java-nio-example/): How to add GCS NIO after the fact.
```

Comment:
OK, I put "legacy code" then (the distinction is whether you are recompiling or not - adding the gcloud-java-nio dependency to existing code would fall under "case 1" i.e. what gcloud-java-examples illustrates).